### ISSUE TITLE
Accessibility: The Search Form in the footer is missing an adequate form label

### ISSUE THREAD

**Issue by possumbilities** [creativecommons/vocabulary#384]
## Description

A form control does not have a corresponding label. 

Specifically:

`````
<input type="text" value="" name="s" class="" id="s" placeholder="Search">
`````

## Expectation

It should have an appropriate label.

## Resolution

- [x] I would be interested in resolving this bug.
